mlir/lib/Dialect/ArmSME/Transforms/EnableArmStreaming.cpp

```diff
@@ -61,7 +63,9 @@ struct EnableArmStreamingPass
     this->enableZA = enableZA;
   }
   void runOnOperation() override {
-    std::string attr;
+    if (getOperation()->getAttr(kArmEnableStreamingIgnore))
+      return;
```

Comment:
Please could you add a test for this in `mlir/test/Dialect/ArmSME/enable-arm-streaming.mlir`